webapp/package-lock.json

```diff
@@ -87,10 +90,13 @@
         "@types/react-window": "1.8.6",
         "@types/redux-logger": "3.0.10",
         "@types/swagger-ui-react": "4.18.1",
+        "@types/testing-library__jest-dom": "6.0.0",
         "@types/uuid": "9.0.4",
         "@typescript-eslint/eslint-plugin": "6.14.0",
         "@typescript-eslint/parser": "6.14.0",
         "@vitejs/plugin-react-swc": "3.5.0",
+        "@vitest/coverage-v8": "^1.6.0",
```

Comment:
Remove all `^`

Reply:
its not recommended to manually edit `package-lock.json`, I will try re-run npm install

Reply:
Yes I comment the wrong file